hugo/content/research/2014/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "DORA Research: 2014"
+date: 2024-07-29
+draft: false
+research_year: "2014"
+type: research_archives
+layout: single
+---
+
+[![2014 State of DevOps Report](2014-state-of-devops-report.png)](2014-state-of-devops-report.pdf)
+
+The 2014 State of DevOps Report provides compelling evidence that DevOps is not just a set of technical practices but a cultural shift that can drive significant improvements in IT and organizational performance. By adopting DevOps practices and fostering a culture of collaboration and continuous improvement, organizations can achieve higher levels of productivity, profitability, and market share.
+
+Key findings of the report include:
+* High-performing IT organizations were twice as likely to exceed their profitability, market share, and productivity goals.
+* DevOps practices, such as version control and continuous delivery, are correlated with high IT performance.
+* Organizational culture is a strong predictor of IT and organizational performance, with high-trust cultures leading to better results.
+* Job satisfaction is the number one predictor of organizational performance and is highly correlated with DevOps practices and culture.
+* Continuous integration and continuous delivery, along with automated testing and version control, are key practices that impact IT * performance.
```

Comment:
This line has a spurious asterisk

Reply:
Resolved in 6cc8ec219936bd9ed70e1d469af471d6e46a2ace